libs/localization/src/lib/useNamespaces.tsx

```diff
@@ -11,8 +11,8 @@ export function useNamespaces(namespaces?: string | string[]) {
   } = useContext(LocaleContext)
 
   useEffect(() => {
-    if (!isEmpty(namespaces)) {
-      loadMessages(namespaces as string | string[])
+    if (namespaces && !isEmpty(namespaces)) {
+      loadMessages(namespaces)
```

Comment:
Shouldn't we filter out `null` here rather than changing the signature of `loadMessages` to expect `null[]`?

Reply:
Tes that's true, it would be more logic, let me fix that, thanks :+1: